.vscode/settings.json

```diff
@@ -5,6 +5,9 @@
         120
     ],
   },
+  "[arb]": {
+    "editor.tabSize": 2,
```

Comment:
Is it? If you look at the diff, the arb file was very inconsistently formatted, so I thought it would be easiest to just fix the tab size and be done with it.